src/build-data/os/emscripten.txt

```diff
@@ -15,3 +17,8 @@ filesystem
 dev_random
 posix1
 </target_features>
+
+<aliases>
+emscripten
```

Comment:
Why the self-alias?

Reply:
Otherwise the [`.matches_name("emscripten")`](https://github.com/randombit/botan/pull/2864/files#diff-4d5f3192809ec1b9add6b33007e0c50031ad9a0a2f3f55a481b506468824db2cR2161) didn't work.